bigchaindb/common/schema/transaction.yaml

```diff
@@ -154,7 +154,10 @@ definitions:
             additionalProperties: true
           uri:
             type: string
-            pattern: "^ni:///sha-256;([a-zA-Z0-9_-]{0,86})?(.+)$"
+            pattern: "^ni:///sha-256;([a-zA-Z0-9_-]{0,86})[?]\
```

Comment:
Not sure if this is overkill but we could end the regex with `(?!&)$` just to make sure there will be no `&` in the end.